pybamm/expression_tree/unary_operators.py

```diff
@@ -1014,6 +1014,46 @@ def _sympy_operator(self, child):
             return sympy.Symbol(latex_child)
 
 
+class BoundaryCellValue(BoundaryOperator):
```

Comment:
It seems odd to me that this is part of `unary_operators` rather than being part of a spatial method. can we talk more about how the problem is posed for the moving boundary?

Reply:
Ok, I see why this might be here. It's essentially `BoundaryValue` but taking cell centre rather than edge. I still kind of feel like this is a concern of spatial methods. Let's talk more.